scripts/oracle-setup/src/handlers.ts

```diff
@@ -0,0 +1,176 @@
+/**
+ * Helper functions to handle the oracle setup.
+ */
+import { ApiPromise, Keyring } from "@polkadot/api";
+import { getNewConnection } from "./utils/connectionHelper";
+import { sendAndWaitForSuccess } from "./utils/polkadotjs";
+import { u32, u128, Bytes, Text } from "@polkadot/types-codec";
+import { AnyNumber, AnyString } from "@polkadot/types-codec/types";
+import { AccountId32, Percent } from "@polkadot/types/interfaces";
+import { AddressOrPair } from "@polkadot/api/types";
+import { stringToHex } from "@polkadot/util";
+import { expect } from "chai";
+import { KeyringPair } from "@polkadot/keyring/types";
+
+/**
+ * Establishes connection.
+ *
+ * @returns {
+ * api: ApiPromise,
+ * keyring: Keyring,
+ * walletAlice: KeyringPair,
+ * }
```

Comment:
I don't believe you need the jsdoc comments and the ts types(?) just one should work